lib/Service/Sync/ImapToDbSynchronizer.php

```diff
@@ -202,10 +210,6 @@ public function sync(Account $account,
 		if ($mailbox->getSelectable() === false) {
 			return $rebuildThreads;
 		}
-
-		$client = $this->clientFactory->getClient($account);
-		$client->login(); // Need to login before fetching capabilities.
```

Comment:
… should go back here. The login outside seems arbitrary, so the next developer might drop it.

1. Move the login back to where it matters
2. Do not drop the comment explaining why we need the explicit login